src/stream_management/mod_stream_management.erl

```diff
@@ -284,6 +284,7 @@ foreign_event(Acc, #{c2s_data := StateData,
                      event_type := {call, From},
                      event_tag := ?MODULE,
                      event_content := {resume, H}}, _Extra) ->
+    %% XXX Do we deregister from SM correctly?
```

Comment:
This comment looks like a debug leftover, especially the "XXX". Please rewrite or remove.